drivers/wifi/nrf700x/src/wifi_util.c

```diff
@@ -916,12 +876,6 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 		      nrf_wifi_util_set_he_ltf_gi,
 		      2,
 		      0),
-	SHELL_CMD_ARG(rts_threshold,
```

Comment:
This would be a breaking change in case someone relies on this, update migration guide.

Reply:
Updated "release-notes-changelog.rst"